.github/workflows/build.yml

```diff
@@ -24,12 +55,98 @@ jobs:
           python-version: "3.8"
       - uses: pre-commit/action@v2.0.0
 
-  build:
+  build-windows:
+    needs:
+      - init
     strategy:
+      matrix:
+        os: ${{ fromJSON(needs.init.outputs.build_windows) }}
+        python-version: ${{ fromJSON(needs.init.outputs.python_versions) }}
```

Comment:
python-version on the left and python_version**s** on the right, that is not ideal

Reply:
fixed